recipes/rtg-tools/build.sh

```diff
@@ -16,7 +16,7 @@ cp -rvp scripts $TGT
 echo "RTG_TALKBACK=true     # Attempt to send crash logs to realtime genomics, true to enable
 RTG_USAGE=false   # Enable simple usage logging, true to enable
 RTG_JAVA_OPTS=    # Additional arguments passed to the JVM
-RTG_JAVA=/opt/anaconda1anaconda2anaconda3/bin/java   # point to anaconda installed Java
+RTG_JAVA=\${JAVA_HOME}/bin/java   # point to the installed Java
```

Comment:
I tried installing the artifact, and with `openjdk=23.0.2=hfb9339a_1` installed, `java` gets installed to `$CONDA_PREFIX/lib/jvm/bin/java` not `$CONDA_PREFIX/bin/java`.

It's better if we use [`\${JAVA_HOME}`](https://github.com/conda-forge/openjdk-feedstock/blob/9a7b9510a9aa0fa579d845e6b8328605a3ffea92/recipe/scripts/activate.sh#L6) like [so many other recipes do here](https://github.com/search?q=repo%3Abioconda%2Fbioconda-recipes%20JAVA_HOME&type=code)